backend/curriculum_tracking/serializers.py

```diff
@@ -221,8 +222,12 @@ class Meta:
             "open_pr_count",
             "oldest_open_pr_updated_time",
             "repo_url",
+            "flavour_names"
         ]
 
+    due_time = serializers.DateTimeField()
```

Comment:
Why are you changing this?

Reply:
This change was made before I took this over.